core/src/main/java/org/everit/json/schema/Visitor.java

```diff
@@ -177,6 +181,12 @@ void visitPatternProperties(Map<Regexp, Schema> patternProperties) {
         }
     }
 
+    void visitDefinitionSchemas(Map<String, Schema> definitionSchemas) {
```

Comment:
Are you sure this makes sense? It may trigger unnecessary validation. For example if the schema is 
```json
{
   "type": "object",
   "properties": {
      "A": { "type": "string" }
   },
   "required": ["A"']
   "$defs": {
      "A": { "type": "integer" }
   }
}
```
Then I suspect no instance would pass the validation (the "A" prop would be validated against both `{"type":"string"}` and `{ "type": "integer" }`).